doc/source/docs/howto_build_docs.rst

```diff
@@ -58,18 +58,28 @@ new virtual environment is recommended.
 Dependencies
 ^^^^^^^^^^^^
 
-All of the necessary dependencies for building the NumPy docs can be installed
-with::
+All of the necessary dependencies for building the NumPy docs except for
+Doxygen_ can be installed with::
 
     pip install -r doc_requirements.txt
 
-We currently use Sphinx_ for generating the API and reference
-documentation for NumPy. In addition, building the documentation requires
-the Sphinx extension `plot_directive`, which is shipped with
+We currently use Sphinx_ along with Doxygen_ for generating the API and
+reference documentation for NumPy. In addition, building the documentation
+requires the Sphinx extension `plot_directive`, which is shipped with
 :doc:`Matplotlib <matplotlib:index>`. We also use numpydoc_ to render docstrings in
 the generated API documentation. :doc:`SciPy <scipy:index>`
 is installed since some parts of the documentation require SciPy functions.
 
+For installing Doxygen_, please check the official
+`download <https://www.doxygen.nl/download.html#srcbin>`_ and
+`installation <https://www.doxygen.nl/manual/install.html>`_ pages, or if you
+are using Linux then you can install it through your distribution package manager.
+
+.. note::
+
+   Try to install a newer version of Doxygen_ > 1.8.10 otherwise you may get some
```

Comment:
AFAIK Doxygen is pretty finicky between versions, changes are not always backward compatible; would make sense to require the exact version.

Reply:
hard to be obeyed, since many Linux distros still support old versions of Doxygen 1.6.x.x. However, I think old versions would still be compatible since HTML generator is off anyway, remained warnings are usually related to default behaviors or non-exist config options which can be ignored. Let us keep it like that until be tested in the wild.